app/uk/gov/hmrc/serviceconfigs/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -18,23 +18,28 @@ package uk.gov.hmrc.serviceconfigs.connector
 
 import play.api.Logging
 import play.api.cache.AsyncCacheApi
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
 import play.api.libs.json.Reads
 import uk.gov.hmrc.http.client.HttpClientV2
 import uk.gov.hmrc.http.{HeaderCarrier, StringContextOps}
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 import uk.gov.hmrc.serviceconfigs.model.{ServiceType, Tag, TeamName}
 
+import java.time.Instant
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 object TeamsAndRepositoriesConnector {
-  case class Repo(name: String)
+  case class Repo(name: String, teamNames: Seq[String], endOfLifeDate: Option[Instant])
```

Comment:
Can we introduce `case class RepoName(name: String) extends AnyVal` to not loose the type safety we currently have